src/TestableHttpClient/Response/StatusCodeResponse.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace TestableHttpClient.Response;
+
+internal class StatusCodeResponse : ResponseBase
```

Comment:
Do we really need this class? Can't we make the base class non abstract and just use that?